package.json

```diff
@@ -1,45 +1,40 @@
 {
-  "name": "project_base",
+  "name": "sustainability_kiosk",
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "serve": "SET NODE_OPTIONS=--openssl-legacy-provider && vue-cli-service serve --open",
-    "build": "node --max_old_space_size=4096 node_modules/.bin/vue-cli-service build --mode=production",
-    "build-stage": "node --max_old_space_size=4096 node_modules/.bin/vue-cli-service build --mode staging",
-    "lint": "vue-cli-service lint",
-    "prettier": "prettier --write **/*.{js,ts,tsx,json,html,md,css,scss,vue}",
+    "dev": "vite",
+    "build": "vite build",
+    "build-stage": "vite build --mode staging",
+    "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix src",
+    "prettier": "prettier --write **/*.{js,json,html,md,css,scss,vue}",
     "format": "npm-run-all --continue-on-error prettier \"lint -- --fix\""
   },
   "dependencies": {
-    "axios": "^0.19.2",
+    "autoprefixer": "^10.4.20",
+    "axios": "^1.7.9",
     "cache-loader": "^4.1.0",
-    "core-js": "^2.6.5",
-    "element-ui": "^2.13.1",
+    "element-plus": "^2.9.1",
     "prettier": "^2.7.1",
-    "sass": "^1.26.10",
-    "sass-loader": "^8.0.2",
-    "vue": "^2.6.10",
-    "vue-i18n": "^8.17.3",
-    "vue-router": "^3.1.6",
-    "vuex": "^3.2.0",
+    "vue": "^3.5.13",
+    "vue-i18n": "^9.14.2",
+    "vue-router": "^4.5.0",
+    "vuex": "^4.1.0",
     "xml2js": "^0.6.2"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "^3.12.0",
-    "@vue/cli-plugin-eslint": "^3.12.0",
-    "@vue/cli-service": "^3.12.0",
-    "babel-eslint": "^10.0.1",
-    "eslint": "^5.16.0",
+    "@vitejs/plugin-vue": "^1.6.1",
+    "eslint": "^8.57.1",
     "eslint-config-standard": "^12.0.0",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-node": "^8.0.1",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
     "eslint-plugin-strict-vue": "^1.0.0",
-    "eslint-plugin-vue": "^5.2.3",
+    "eslint-plugin-vue": "^8.7.1",
     "npm-run-all": "^4.1.5",
-    "style-resources-loader": "^1.3.3",
-    "vue-template-compiler": "^2.6.10"
+    "sass": "^1.26.10",
+    "vite": "^2.5.4"
```

Comment:
I think we should be able to update to the newest version of `vite` without any issues (we might need to update a package or two) and that should remove some more vulnerabilities (potentially all of the rest of them based on the `npm audit` output)